sdk/python/tests/integration/feature_repos/universal/data_sources/file.py

```diff
@@ -362,33 +365,34 @@ def create_offline_store_config(self):
 class RemoteOfflineStoreDataSourceCreator(FileDataSourceCreator):
     def __init__(self, project_name: str, *args, **kwargs):
         super().__init__(project_name)
-        self.server_port: int = 0
+        self.server_port: int = 8815
         self.proc: Process = None
 
     def setup(self, registry: RegistryConfig):
         parent_offline_config = super().create_offline_store_config()
 
-        fs = FeatureStore(
-            config=RepoConfig(
-                project=self.project_name,
-                provider="local",
-                offline_store=parent_offline_config,
-                registry=registry.path,
-                entity_key_serialization_version=2,
-            )
-        )
         self.server_port = free_port()
         host = "0.0.0.0"
         self.proc = Process(
-            target=start_server,
-            args=(fs, host, self.server_port),
+            target=start_test_local_server,
+            args=(
+                parent_offline_config,
+                self.project_name,
+                registry.path,
+                host,
+                self.server_port,
+            ),
         )
         self.proc.start()
+        _time_out_sec: int = 180
+
         # Wait for server to start
         wait_retry_backoff(
             lambda: (None, check_port_open(host, self.server_port)),
-            timeout_secs=10,
+            timeout_secs=_time_out_sec,
+            timeout_msg=f"Unable to start the feast server in {_time_out_sec} seconds for remote offline store type, port={self.server_port}",
         )
+        logger.info("Port has been freed.")
```

Comment:
isn't this "port has been occupied" instead?

Reply:
this code likely be triggered multiple times for each test case, isn't it a good idea to add the port number also to log just to differentiate?